### PR TITLE
New vpc nat

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: nsxt/resource_nsxt_vpc_ip_address_allocation_test.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+		},
+	})
+}
+
+func TestAccResourceNsxtVpcIpAddressAllocation_importBasic(t *testing.T) {
+	name := getAccTestResourceName()
+	testResourceName := "nsxt_vpc_ip_address_allocation.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
```
Comment (ksamoray) [inline nsxt/resource_nsxt_vpc_ip_address_allocation_test.go:88]: Use `testAccOnlyVPC(t)`

**Review by ksamoray — COMMENTED** [review]
Documentation, and DS test are planned for later?

File: nsxt/resource_nsxt_vpc_ip_address_allocation_test.go [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  description     = "%s"
+  allocation_size = %s
+  tag {
+    scope = "scope1"
+    tag   = "tag1"
+  }
+}`, testAccNsxtPolicyMultitenancyContext(), attrMap["display_name"], attrMap["description"], attrMap["allocation_size"])
+}
+
+func testAccNsxtVpcIpAddressAllocationMinimalistic() string {
```
[the remark this answers is not in the document]
Comment (annakhm) [reply nsxt/resource_nsxt_vpc_ip_address_allocation_test.go:175]: The first one will evolve once more support from NSX comes our way

**Review by ksamoray — APPROVED** [review]

**Pull request merged**